index.js

```diff
@@ -99,7 +103,10 @@ function resolveUrlLoader(content, sourceMap) {
   if (useMap) {
 
     // source-map sources seem to be relative to the file being processed
-    absoluteToRelative(reworked.map.sources, filePath);
+    absoluteToRelative(reworked.map.sources, path.resolve(filePath, sourceRoot));
```

Comment:
I like terse code but I am not familiar with using `path.resolve` in this particular way.

When `sourceRoot === undefined` (i.e. no source map implies uninitialised) is the result equal to `filePath`?